### DISCUSSION TITLE
Is there support for Android native widgets?

### DISCUSSION THREAD

**Discussion by Raj2032** [tauri-apps/tauri#3122] [General]
Hi, I was wondering if there was support for Android native widgets? If not then if I can use its material design from a third party library and if this is possible how would I import a third party library?

**Comment by amrbashir** [comment]
Android support or mobile in general is not possible yet but once it lands, there will be no native widget at all; it will be just an Android app that hosts one Webview. You can use CSS or use a material css framework to replicate the style of native Android widgets.

**Reply by Raj2032** [reply]
I see thanks mate.
